ios/RNSScreen.h

```diff
@@ -109,12 +112,16 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)notifyDisappear;
 - (void)updateBounds;
 - (void)notifyDismissedWithCount:(int)dismissCount;
+- (instancetype)initWithFrame:(CGRect)frame;
```

Comment:
Are we using this initializer somewhere?

Reply:
No, but it should've been added there before as it is an initialiser and comes handy in various scenarios to be able to init the screen (especially when debugging). Also it is planned to separate `RNSScreenView` impl from `RNSScreen` & `RNSScreenViewManager` thus it will be necessary change. 

I could settle for introducing this change in separate PR, but when commiting I initially decided to leave this one-liner as no-harmer.